starknet_py/net/schemas/rpc.py

```diff
@@ -288,79 +283,6 @@ def make_dataclass(self, data, **kwargs) -> BlockStateUpdate:
         )
 
 
-class StructMemberSchema(Schema):
```

Comment:
Old abi schemas has been removed, but I don't see them being replaced. Shouldn't you also change the code in FullNodeClient or some existing rpc schemas to use newly defined ABIs?

Reply:
https://github.com/software-mansion/starknet.py/issues/512